.circleci/config.yml

```diff
@@ -339,8 +339,8 @@ commands:
     steps:
       - run:
           name: run tests
-          command: mvn -B org.jacoco:jacoco-maven-plugin:report org.jacoco:jacoco-maven-plugin:report-aggregate $MAVEN_GOAL -P$TESTING_PROFILE,coverage -ntp
-
+          command: mvn -B org.jacoco:jacoco-maven-plugin:report org.jacoco:jacoco-maven-plugin:report-aggregate $MAVEN_GOAL -P$TESTING_PROFILE,coverage -ntp | grep  -v "^Running Changeset:"
```

Comment:
If there's an error applying a change set (rare, I know), will this suppress the error message as well?

Reply:
As far as I can tell whenever there is an error when applying a change set one of two things happens,

1. The outputted line does not start with `Running Changeset` and instead starts with `ERROR` (meaning it will not be removed), such was the case in this error example,
 ```
ERROR [2022-07-21 23:50:20,831] liquibase.parser.core.xml.XMLChangeLogSAXParser$1: cvc-complex-type.2.4.a: Invalid content was found starting with element ...
 ```
2. At least two lines are outputted, the first one starts with `Running Changeset` (which will be removed), but the following lines explain the error and provide context (which aren't cropped out), such was the case in this error example,

```
Running Changeset: home/fhembroff/Documents/Dockstore3/dockstore-cli/dockstore-cli-integration-testing/../dockstore-webservice/src/main/resources/migrations.test.confidential2.xml::1511391261451-3::dyuen (generated)
liquibase.exception.LiquibaseException: liquibase.exception.MigrationFailedException: Migration failed for change set home/fhembroff/Documents/Dockstore3/dockstore-cli/dockstore-cli-integration-testing/../dockstore-webservice/src/main/resources/migrations.test.confidential2.xml::1511391261451-3::dyuen (generated):
     Reason: liquibase.exception.DatabaseException: ERROR: column "blue" does not exist
```

Either way information about the error and it's context is displayed. So I don't think this will affect our ability to debug changeset error messages.

\* NOTE: In both code example there was significantly more output.